src/main/kotlin/org/jmailen/gradle/kotlinter/support/KtLintEditoConfigOverrides.kt

```diff
@@ -0,0 +1,16 @@
+package org.jmailen.gradle.kotlinter.support
+
+import com.pinterest.ktlint.core.api.DefaultEditorConfigProperties.disabledRulesProperty
+import com.pinterest.ktlint.core.api.EditorConfigOverride
+import com.pinterest.ktlint.core.api.EditorConfigOverride.Companion.emptyEditorConfigOverride
+import com.pinterest.ktlint.core.api.EditorConfigOverride.Companion.plus
+
+internal fun editorConfigOverride(ktLintParams: KtLintParams): EditorConfigOverride {
+    val rules = ktLintParams.disabledRules
+
+    return if (rules.isEmpty()) {
+        emptyEditorConfigOverride
+    } else {
+        EditorConfigOverride.from(disabledRulesProperty to rules.joinToString(separator = ","))
+    }
```

Comment:
I followed the migration guide from [the release notes](https://github.com/pinterest/ktlint/releases/tag/0.46.0)